<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="labelView">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>551</width>
     <height>391</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSliderContrast">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>11</y>
     <width>28</width>
     <height>351</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>33</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSliderBrightness">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>11</y>
     <width>28</width>
     <height>351</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>379</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>对比度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>379</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>亮度</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
